src/backend/SimpleCache.h

```diff
@@ -45,6 +46,9 @@ class SimpleCache
     std::optional<LedgerObject>
     getPredecessor(ripple::uint256 const& key, uint32_t seq) const;
 
+    void
+    setDisabled();
```

Comment:
are we actually calling this anywhere?

Reply:
We are _now_

Reply:
Sorry, accidentally `git stash` the function call.